scripts/test-install-release-filter.sh

```
@@ -973,6 +973,28 @@ test_git_bash_latest_installs_windows_zip_asset() {
   assert_contains "$work_dir/output.txt" "uloop mock version"
 }
 
+# Verifies -UseBasicParsing on the two payload downloads, the PS 5.1-only
+# TLS 1.2 -bor assignment, and that that assignment appears before the first
+# Invoke-WebRequest in install.ps1.
+test_powershell_installer_enables_tls12_and_basic_parsing() {
+  assert_contains "$ROOT_DIR/scripts/install.ps1" 'if ($PSVersionTable.PSVersion.Major -lt 6) {'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)